hknweb/candidate/models.py

```diff
@@ -56,35 +55,25 @@ def rejected(self):
     def __str__(self):
         return self.name
 
-class CandidateForm(models.Model):
```

Comment:
What's the reason for this move? Doesn't matter too much, just curious lol

Reply:
Candidate forms seem less related to challenges and bitbyte so I moved it down